src/components/Item.jsx

```diff
@@ -51,13 +54,43 @@ const Item = ({
     return estimatedInterval;
   };
 
+  function handleClick(e) {
+    e.preventDefault();
+    const confirmation = window.confirm(
+      `Are you sure you want to delete ${name}?`,
+    );
+
+    if (confirmation) {
+      db.collection('items')
+        .doc(id)
+        .delete()
+        .then(() => {
+          setActionMessage(`${name} successfully deleted!`);
+          setTimeout(() => {
+            setActionMessage(null);
+          }, 10000);
+        })
+        .catch((error) => {
+          console.error('Error removing document: ', error);
+        });
+    }
+  }
+
   const checked = checkDate(lastPurchasedDate);
   const className = checked ? 'checked' : '';
   return (
-    <div className="check-item">
-      <input type="checkbox" onChange={checkHandler} checked={checked} />
-      <li className={className}>{name}</li>
-    </div>
+    <li className={`${className} check-item`}>
+      <div>
+        <input type="checkbox" onChange={checkHandler} checked={checked} />
+        {name}
+      </div>
+
+      <div>
+        <a className="delete-button" href="x" onClick={handleClick}>
```

Comment:
What is this `href="x"` doing?

Reply:
Hi Kevin href doesn't seem to work without content (and initially we thought a link would be best), but has been removed now.